jarbas/core/management/commands/reimbursements.py

```diff
@@ -31,53 +34,31 @@ def handle(self, *args, **options):
     @property
     def reimbursements(self):
         """Returns a Generator with a dict object for each row."""
-        with lzma.open(self.path, mode='rt') as file_handler:
-            for row in csv.DictReader(file_handler):
-                yield self.serialize(row)
-
-    def serialize(self, reimbursement):
-        """Read the dict generated by DictReader and fix content types"""
-
-        missing = ('probability', 'suspicions')
-        for key in missing:
-            reimbursement[key] = None
-
+        force_types = {
+            'cnpj_cpf': TextField,
+            'document_number': TextField,
+            'leg_of_the_trip': TextField,
+            'congressperson_id': IntegerField,
+            'congressperson_document': IntegerField,
```

Comment:
If the document starts with 0 you'll loose this information after converting to `int` (`int('00001')` = `1`), so you may want a `TextField` here (same for other document fields).

Reply:
That makes sense but as those fields are integer in the database (we checked, no leading zeros so far) I decided to define `rows` fields coherent to what the model expects…